core/src/main/java/io/confluent/kafka/schemaregistry/storage/SchemaValue.java

```diff
@@ -92,6 +96,16 @@ public void setSchema(String schema) {
     this.schema = schema;
   }
 
+  @JsonProperty("deleted")
+  public boolean isDeleted() {
+    return deleted;
+  }
+
+  @JsonProperty("deleted")
```

Comment:
Ah, never mind, I see that the constructor is fine as long as we can also use `setDeleted`. I might still update the constructor since it's a bit confusing to have different fields set via different methods.